server/game/sprites/player.js

```diff
@@ -75,7 +75,7 @@ class Player extends Phaser.Physics.Arcade.Sprite {
     }
 
     if (onFloor  || this.body.touching.down ) {
-      this.jumpCount = -1
+      this.jumpCount = 0
```

Comment:
I changed this again because you should only be able to jump twice. When it was -1, you could do it 3 times.

Reply:
Ah I see, that makes sense.